changes-entries/http1_separation.txt

```diff
@@ -0,0 +1,31 @@
+  *) mod_http: separating the HTTP semantic handling from the HTTP/1.x
+     serialization (transcoding) handling by splitting filters.
+     - new REQUEST/RESPONSE/HEADERS bucket types that carry the
+       meta information about a HTTP request or response or trailers.
+       TRANSCODE input filters convert protocol version specific
+       bytes to REQUEST/HEADER buckets.
+       TRANSCODE output filters convert REPONSE/HEADER buckets into
+       the byte format suitable for the HTTP protocol version.
+     - new module mod_http1 that has the new transcoding filters.
+       also linked statically, just like mod_http.
+     - "chunked" encoding handled in mod_http1 TRANSCODE filters.
+     - Response header check and completion done in mod_http
+       filter for all HTTP procotol versions.
+     - ap_send_interim_response() now generates a RESPONSE bucket
+       and sends it through the protocol output filters of a request.
+     - there are still several code locations in the server where
+     - several "Transfer-Encoding" checks have been moved from
+       generic HTTP handling into mod_http1.
+       A new note AP_NOTE_REQUEST_BODY_INDETERMINATE has been
+       introduced that, when present in r->notes, indicates that
+       the length of a request body is not known.
```

Comment:
Maybe a new request_rec `body_indeterminate` field/flag rather than a `notes`?
(notes are not very effectively accessed, so it depends on how often it needs to be known by filters/handlers)

Reply:
Agreed. I started the note early when I did not want to mess with request_rec.

Reply:
Added a field to `request_rec` named `body_indeterminate` and removed all occurences of `AP_NOTE_REQUEST_BODY_INDETERMINATE`.